src/core/src/op/util/node_util.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/util/node_util.hpp"
+
+namespace ov {
+namespace op {
+namespace util {
+void set_name(ov::Node& node, const std::string& name, size_t output_port) {
+    node.set_friendly_name(name);
+    node.get_output_tensor(output_port).set_names({name});
```

Comment:
Ticket mention that some generalization should be applied. 
Consider add possibility to set names to all outputs?

Example: make some special value like `-1` or `size_t::max` as `all ports` and set names on output tensor like 
`name.idx`, for single output port it will just add suffix for specified output and not change others.